dynalab/dockerfiles/prod/Dockerfile

```diff
@@ -24,15 +24,15 @@ RUN apt-get update && \
     curl \
     vim \
     git \
-    && rm -rf /var/lib/apt/lists/* 
```

Comment:
Same thing as before here (maybe we can remove the space in as it does mess up something, although very unlikely that it does)

Reply:
Should be fine - thanks for catching though.